client/client.go

```diff
@@ -386,8 +398,8 @@ func (c *Client) Stats() map[string]map[string]string {
 	stats := map[string]map[string]string{
 		"client": map[string]string{
 			"node_id":         c.Node().ID,
-			"known_servers":   toString(uint64(c.rpcProxy.NumServers())),
-			"num_allocations": toString(uint64(numAllocs)),
+			"known_servers":   strconv.Itoa(len(c.servers.all())),
```

Comment:
(I'd vote for replace existing, but I don't know if people rely on this to be stable between releases)